packages/components/stories/tabs/stories/fited-tabs.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { boolean } from '@storybook/addon-knobs';
 import Tabs from 'Components/tabs';
-import Wrapper from '../../shared/wrapper';
+import Wrapper from '../../shared/wrapper.jsx';
```

Comment:
@jim-deriv why are adding .jsx extension. Shouldn't it be resolved automatically?

Reply:
Hello @balakrishna-binary,

Due to eslint configurations, this was enforced and I was hesitant to change the configs, but I have disabled that config in eslintrc